tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -686,7 +686,7 @@ declare namespace ts {
         readonly kind: SyntaxKind.ConstructSignature;
     }
     export type BindingName = Identifier | BindingPattern;
-    export interface VariableDeclaration extends NamedDeclaration {
+    export interface VariableDeclaration extends NamedDeclaration, JSDocContainer {
```

Comment:
@sandersn is the best person to judge if this is appropriate

Reply:
@DanielRosenwasser Yes, you are right. Quick note: `try {} catch (/** @type {unknown} */ e) {}` - `e` is a `VariableDeclaration`, and in the current implementation, JSDoc comment is not attached to `e`. To have the opportunity to add type declaration we node has to have JSDoc, these changes related to that. Does that make sense?  If there is a better way to do this, I can change that.

Reply:
No, I think this is the best way.
In fact, in normal `var` declarations, we don't currently look for jsdoc on VariableDeclarations, meaning that `x: boolean` and `y: boolean` below, instead of `string` and `number` (or error, because of the conflict).

```js
/** @type {boolean} */
var /** @type {string} */ x, /** @type {number} */ y;
```

Actually, can you add the above as a test? I wonder how much of that will Just Work after this change. It's likely that quite a bit of it will, because there's no walking up the tree required.

If it *doesn't*, that's fine; we haven't had anybody ask for it, nor have we gone looking whether it's a common pattern (my guess is No).

Reply:
> Actually, can you add the above as a test? 

@sandersn I've added a test and the other tests seem to work fine.